srcpkgs/espeak-ng/template

```diff
@@ -0,0 +1,72 @@
+# Template file for 'espeak-ng'
+pkgname=espeak-ng
+version=1.50
+revision=1
+build_style=gnu-configure
+make_build_args="-j1"
+hostmakedepends="automake libtool which pkg-config"
+makedepends="pcaudiolib-devel libsonic-devel"
+short_desc="Text to Speech engine"
+maintainer="Joey <joey@imap.cc>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/espeak-ng/espeak-ng/"
+distfiles="https://github.com/espeak-ng/espeak-ng/archive/${version}.tar.gz"
+checksum=5ce9f24ee662b5822a4acc45bed31425e70d7c50707b96b6c1603a335c7759fa
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" espeak-ng"
+fi
+
+pre_configure() {
+	./autogen.sh
+}
+
+post_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		sed -i '2785,2818 s_src/espeak-ng_/usr/bin/espeak-ng_' Makefile
+		sed -i '2819,$ s_../src/espeak-ng_/usr/bin/espeak-ng_' Makefile
+				sed -i '3244,$ s_src/espeak-ng_/usr/bin/espeak-ng_' Makefile
+	fi
```

Comment:
I guess just putting the files there didn't work? This is quite unfortunate.

Having numbered seds is quite fragile, though, so ideally we could come up with a patch for it.

Also, indentation of the last `sed` line is too far right by 2 tabs.